tests/test_optimizer_random_dag.py

```diff
@@ -140,6 +153,9 @@ def compare_optimization_results(dag: sky.Dag, minimize_cost: bool):
         objective = sky.Optimizer._compute_total_time(dag.get_graph(),
                                                       dag.tasks, optimizer_plan)
 
+    print('=== optimizer plan ===', file=sys.stderr)
+    print(optimizer_plan, file=sys.stderr)
+    print('=== brute force ===', file=sys.stderr)
```

Comment:
Do we want to retain these print statements? If so, should we consider something like this for readability (I wasn't sure where the brute force print was coming from):

```
    min_objective, bf_plan = find_min_objective(copy_dag, minimize_cost)

    print('=== optimizer plan ===', file=sys.stderr)
    print(optimizer_plan, file=sys.stderr)
    print('=== brute force ===', file=sys.stderr)
    print(bf_plan, file=sys.stderr)
```